From 989adafd61df22e033ee361ea8062063b94724ee Mon Sep 17 00:00:00 2001
From: SunnyQeen <sunnyqeen@gmail.com>
Date: Tue, 12 Nov 2024 21:50:10 +0100
Subject: disable sockdiag


diff --git a/configure b/configure
index 36c6ad67..23870fc3 100644
--- a/configure
+++ b/configure
@@ -18681,7 +18681,7 @@ then :
   cat >>confdefs.h <<_ACEOF
 #define `printf "%s\n" "HAVE_$ac_header" | $as_tr_cpp` 1
 _ACEOF
- sockdiag_header=yes
+ sockdiag_header=no
 else $as_nop
   sockdiag_header=no
 fi
